src/js/components/Button/stories/typescript/Basic.tsx

```diff
@@ -1,26 +0,0 @@
-import React from 'react';
-import { storiesOf } from '@storybook/react';
-import isChromatic from 'chromatic/isChromatic';
-
-import { grommet, Box, Button, Grommet } from 'grommet';
-
-const BasicButtons = props => (
```

Comment:
we should also:
1. Remove CustomColor.tsx.
2. Add ThemeType and comments to the story we're keeping

Reply:
Removed `CustomColor.tsx`. For Button `Custom.tsx` I realized that it is different from `Custom.js` so I kept both and renamed the tsx story to TS-Custom and added comments about Typescript. Let me know if I should have done something else here instead.